app/src/main/java/edu/akelael/comics/model/IComicListModel.java

```diff
@@ -0,0 +1,6 @@
+package edu.akelael.comics.model;
+
+public interface IComicListModel {
```

Comment:
Using and I to start an interface name, or using Impl for a class that implements an interface, are usually smells. In MVP code examples, interfaces are usually provided for abstraction, but do they really are adding that value?